src/main/java/com/github/danielflower/mavenplugins/gitlog/renderers/SimpleHtmlRenderer.java

```diff
@@ -73,11 +73,27 @@ public void renderCommit(RevCommit commit) throws IOException {
 
 		tableHtml.append("\t\t<tr>")
 				.append("<td class=\"date\">").append(date).append("</td>")
-				.append("<td>").append(message).append("</td>")
 				.append("<td>").append(authorHtml).append("</td>")
+				.append("<td>").append(message).append("</td>")
 				.append("</tr>").append(NEW_LINE);
 	}
 
+	private String formatLongMessage(String gitMessage) {
+		String lines[] = StringEscapeUtils.escapeHtml4(gitMessage).split(LINE_SEPARATOR);
+		if (lines.length <=1) {
+			return gitMessage;
+		}
+		StringBuilder builder = new StringBuilder();
+		for (int i = 0; i < lines.length; i++) {
```

Comment:
Would it be more straightforward to just wrap every line in a div: 

    "<div class=\"commitDetails\">" + lines[i] + "</div>"